security/pkg/workload/server.go

```diff
@@ -61,12 +49,21 @@ type SDSServer struct {
 
 	// Read/Write mutex for privateKey
 	privateKeyGuard sync.RWMutex
```

Comment:
Can we use KeyCertBundle for better protection of the key and certs? https://github.com/istio/istio/blob/master/security/pkg/pki/util/keycertbundle.go

Reply:
I will do that in another PR